aten/src/ATen/native/cuda/MiscUtils.h

```diff
@@ -0,0 +1,64 @@
+#include "ATen/ATen.h"
+#include "THC.h"  // for USE_MAGMA
+
+#ifdef USE_MAGMA
+#include <magma.h>
+#include <magma_types.h>
+#endif
+
+namespace at {
+namespace native {
+
+#ifdef USE_MAGMA
+
+// RAII for a MAGMA Queue
+struct MAGMAQueue {
+
+  // Default constructor, does nothing.
+  MAGMAQueue() = default;
```

Comment:
Er maybe let's delete this. `MagmaQueue() = delete`;
It doesn't make sense for people to create a MAGMAQueue via the default constructor because then magma_queue_destroy will be called on an uninitialized magma_queue_t in the destructor.

Reply:
Done! Thank you for the suggestion.